content/recipes/swagger.md

```diff
@@ -348,52 +412,219 @@ Let's open the browser and verify the generated `Cat` model:
 
 <figure><img src="/assets/swagger-response-type.png" /></figure>
 
-#### Authentication
+#### Security
+
+To define which security mechanisms should be used for specific operation, use `@ApiSecurity()` decorator.
```

Comment:
To define which security mechanisms should be used for specific operation, use `@ApiSecurity()` decorator.

->

To define which security mechanisms should be used for a specific operation, use the `@ApiSecurity()` decorator.